pom.xml

```diff
@@ -132,15 +123,18 @@
                       <property name="severity" value="error"/>
                       <property name="eachLine" value="true"/>
                     </module>
+                    <module name="SuppressWarningsFilter"/>
                     <module name="TreeWalker">
                       <property name="cacheFile" value="target/cachefile"/>
                       <property name="tabWidth" value="4"/>
                       <!-- Checks for Javadoc comments. -->
                       <module name="JavadocStyle"/>
                       <!-- TODO: Enable NonEmptyAtclauseDescription after https://github.com/checkstyle/checkstyle/issues/1064 is fixed -->
+                      <!-- Update: #1064 is resolved and the performance has greatly improved but this check still adds ~2 seconds -->
+                      <!-- to the build time -> always measure build time before enabling this module (use github.com/jcgay/maven-profiler) -->
                       <!--<module name="NonEmptyAtclauseDescription"/>-->
                       <!-- Checks for imports. -->
-                      <!--<module name="UnusedImports"/>-->
+                      <module name="UnusedImports"/>
```

Comment:
Failing on unused imports is still controversial:
1) It slows down the build apparently
2) The rest of KIE doesn't do this yet:
  https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L495
3) It's easy to fix automatically before a CR1 release etc.
4) It doesn't impact the bits of the .class files.

I don't want to start turning down contributions from contributors because of this, so I'd rather leave this commented out, for now.